### PR TITLE
Release 0.4.0.

### PR DESCRIPTION
**Pull request by jmjoy**

### CODE REVIEW

File: graphicsmagick-sys/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "graphicsmagick-sys"
-version = "0.1.2"
+version = "0.4.0"
```
Comment (NobodyXu) [inline graphicsmagick-sys/Cargo.toml:3]: We might not need a new major release on sys crate since I did not modify any API of it.

**Review by NobodyXu — CHANGES_REQUESTED** [review]

File: graphicsmagick-sys/Cargo.toml (hunk 1 above)
Comment (jmjoy) [reply graphicsmagick-sys/Cargo.toml:3]: But build.rs is changed.

Comment (jmjoy) [reply graphicsmagick-sys/Cargo.toml:3]: I known what you mean, but it has released. :cry:

Comment (NobodyXu) [reply graphicsmagick-sys/Cargo.toml:3]: It should be ok unless one depends on both graphicsmagick-sys 0.2.1 and 0.4.0

**Pull request merged**